js_modules/dagster-ui/packages/ui-core/src/assets/catalog/AssetCatalogTableV2.tsx

```diff
@@ -412,6 +380,35 @@ const Table = React.memo(
               ) : null}
             </Box>
             <Box flex={{direction: 'row', alignItems: 'center', gap: 8}}>
+              <Box flex={{direction: 'row', alignItems: 'center', gap: 4}}>
+                <Body color={Colors.textLight()}>Group by</Body>
+                <Select<(typeof GROUP_BY_ITEMS)[number]>
```

Comment:
It might be nice to pull this Select out as a separate little component since it doesn't require too many props from the surrounding context